SetupScript.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
+if [ $USER -ne "root"]
+	then
+	echo "Please run as root user"
+	exit 1
+fi
+#base script to fresh install systems onto base image for raspberry pi
+
+#check internet connection
+ping -q -c1 google.com > /dev/null
+
+if [ $? -ne 0 ]
+then
+	echo "No Internet Connection...Aborting"
+	exit 1
+fi
+
+#configure displays
+(cd /opt/ && git clone https://github.com/UCSolarCarTeam/Epsilon-Raspberry.git)
```

Comment:
This doens't require a subshell command either, it'd be better to specify the target directory.

Reply:
Above you asked me to change it because it is hard for the reader to keep track of the directories, now you would like it to be changed back, I'm confused about when to create sub-shells.